javascript-forms/form-events.js

```diff
@@ -0,0 +1,24 @@
+const $inp = document.querySelectorAll('input');
+const $textarea = document.querySelector('textarea');
+
+function handleFocus(event) {
+  console.log(`${event.type} event fired`);
+  console.log(`event.target.name: ${event.target.name}`);
+}
+function handleBlur(event) {
+  console.log(`${event.type} event fired`);
+  console.log(`event.target.name: ${event.type}`);
+}
+function handleInput(event) {
+  console.log(`${event.target.name} event fired`);
+  console.log(`event.target.value: ${event.target.value}`);
+}
+
+$inp.forEach((inp) => {
+  $inp.addEventListener('focus', handleFocus);
+  $inp.addEventListener('blur', handleBlur);
+  $inp.addEventListener('input', handleInput);
```

Comment:
This technically needs to be `inp.addEventListener`, since you are looping over and accessing them individually. By doing `$inp`, you are trying to add the event listeners to the `Nodelist`